docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -277,38 +286,164 @@ Scale-up of breast cancer screening coverage among insured population
++--------------------------------------------------------------------------------------------------+
+ 
+ 
+
+.. note:: 
+  see :download:`breast cancer screening memo <breast_cancer_screening_memo.docx>` for more in depth explanation how modelling decisions were adpated from guidelines, as well as assumptions and limitations of these modelling decisions. 
+
+
+:underline:`II. Probability of attending screening`
+
+ - 1) All simulants will be due a screening according to their attributes in the decision tree
+ - 2) Probability of simulants attending their first due screening is 30% (SD=0.3). *Note: this is the parameter we vary in the scale-up scenario* 
+ - 3) If a simulant attends a screening, they have 1.89 (95%CI 1.06-2.49) (Yan et al 2017) more odds of attending their next screening than those who did not attend a screening. 
+
++----------------+-------------+---------------+----------+
+|                | Attended    |Did not attend | Total    |
+|                | last screen |last screen    |          |
++----------------+-------------+---------------+----------+
+| Attends next   |  a          |  b            | a+b      |
+| screen         |             |               |          |
++----------------+-------------+---------------+----------+
+| Does not attend|  c          |  d            | c+d      |
+| next screen    |             |               |          |
++----------------+-------------+---------------+----------+
+|                | a+c         | b+d           | a+b+c+d  |
++----------------+-------------+---------------+----------+ 
+
+
+      (1) :math:`P(\text{attended last screen}) = \frac{a+c}{a+b+c+d}` = 30% (SD 0.3%)
+      (2) :math:`P(\text{attends next screen}) = \frac{a+b}{a+b+c+d}`  = 30% (SD 0.3%)
+      (3) OR = :math:`\frac{a/c}{b/d}=\frac{ad}{bc}` = 1.89 (95%CI 1.06-2.49)
+      (4) a+b+c+d = 1
+
+.. code-block:: Python
+
+  1. Solve for a, b, c, d by first solving the following quadratic equation:
+
+  (OR-1)b^2 + b - P(1-P) = 0 
+
+  Once you obtain b, then
+  | c=b
+  | a=P-b
+  | d=(1-P)-b
+
+  2. Obtain PAF for unmatched case-control OR
+
+  | PAF= 1- [b/(a+b)]/[d/(c+d)]
+  | Error factor of the 1-PAF = exp{1.96 x √[a/(b(a+b)) + c/(d(c+d)]}
+
+Using OR value of 1.89 and P as 0.3
+
+  - a = 0.11912
+  - b = 0.18088
+  - c = 0.18088
+  - d = 0.51912
+  - 1-PAF = 0.813
+  - PAF = 0.187
+  
+
+:math:`P(\text{attended next screen among those who attended last screen}) = \frac{a}{a+c}` = 39.7%
+:math:`P(\text{attended next screen among those who did not attend last screen}) =\frac{b}{b+d}` = 25.8%
+
+:math:`P(\text{attended next screen among those who attended last screen}) = P(1-PAF)*OR = 0.3x0.813x1.89` = 46.1% 
+:math:`P(\text{attended next screen among those who did not attend last screen}) =P(1-PAF) = 0.3x0.813` = 24.4%
+
+.. todo::
+
+    In the above, I presented two ways to calculate the probability of someone going for their next screening, the second one uses the PAF. Not sure which one is correct? 
```

Comment:
I rarely link OR to PAF, but the second equation make sense to me. Would the first one easier for SEs to implement?